psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
Oh and please rename `.psalm.xml.dist` to `psalm.xml` after removing this file. I have to update the step-by-step guide from the issue as I failed there... again 🙈

Reply:
You mighth ave to re-run `vendor/bin/psalm --set-baseline=psalm-baseline.xml` after the renaming

Reply:
@boesing , I did as per your suggestion. Please let me know if its good now?